R/colour-text.r

```diff
@@ -1,8 +1,9 @@
-
+# fix the check NOTE:Namespace in Imports field not imported from: ‘crayon’
```

Comment:
Yes, definitely. Your fix is correct, I was just being nit picky about the R CMD check complaint.